[0.006s] Invoking command in '/home/mk/ros2_study/src/my_first_package_msgs/build/my_first_package_msgs': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/home/mk/ros2_study/install/my_first_package:/opt/ros/humble /usr/bin/cmake --build /home/mk/ros2_study/src/my_first_package_msgs/build/my_first_package_msgs -- -j16 -l16
[0.033s] -- Found ament_cmake: 1.3.10 (/opt/ros/humble/share/ament_cmake/cmake)
[0.117s] -- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
[0.119s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.122s] -- Found rosidl_adapter: 3.1.5 (/opt/ros/humble/share/rosidl_adapter/cmake)
[0.143s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.267s] [31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[0.267s]   execute_process(/usr/bin/python3 -m rosidl_adapter --package-name
[0.267s]   my_first_package_msgs --arguments-file
[0.267s]   /home/mk/ros2_study/src/my_first_package_msgs/build/my_first_package_msgs/rosidl_adapter__arguments__my_first_package_msgs.json
[0.267s]   --output-dir
[0.267s]   /home/mk/ros2_study/src/my_first_package_msgs/build/my_first_package_msgs/rosidl_adapter/my_first_package_msgs
[0.267s]   --output-file
[0.267s]   /home/mk/ros2_study/src/my_first_package_msgs/build/my_first_package_msgs/rosidl_adapter/my_first_package_msgs.idls)
[0.267s]   returned error code 1:
[0.267s] 
[0.267s]   Traceback (most recent call last):
[0.267s] 
[0.267s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[0.267s]       return _run_code(code, main_globals, None,
[0.267s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[0.267s]       exec(code, run_globals)
[0.267s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[0.267s]       sys.exit(main())
[0.267s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[0.267s]       abs_idl_file = convert_to_idl(
[0.267s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 29, in convert_to_idl
[0.267s]       return convert_action_to_idl(
[0.268s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/action/__init__.py", line 28, in convert_action_to_idl
[0.268s]       action = parse_action_string(package_name, input_file.stem, content)
[0.268s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 894, in parse_action_string
[0.268s]       raise InvalidActionSpecification(
[0.268s] 
[0.268s]   rosidl_adapter.parser.InvalidActionSpecification: Number of '---'
[0.268s]   separators nonconformant with action definition
[0.268s] 
[0.268s] Call Stack (most recent call first):
[0.268s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[0.268s]   CMakeLists.txt:16 (rosidl_generate_interfaces)
[0.268s] 
[0.268s] [0m
[0.268s] -- Configuring incomplete, errors occurred!
[0.268s] See also "/home/mk/ros2_study/src/my_first_package_msgs/build/my_first_package_msgs/CMakeFiles/CMakeOutput.log".
[0.271s] gmake: *** [Makefile:963: cmake_check_build_system] Error 1
[0.273s] Invoked command in '/home/mk/ros2_study/src/my_first_package_msgs/build/my_first_package_msgs' returned '2': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/home/mk/ros2_study/install/my_first_package:/opt/ros/humble /usr/bin/cmake --build /home/mk/ros2_study/src/my_first_package_msgs/build/my_first_package_msgs -- -j16 -l16
